### PR TITLE
Added introduction and basic types section for typescript

### PR DESCRIPTION
**Pull request by justsumant**
added introduction and basic types section for typescript

### CODE REVIEW

**Comment by justsumant** [comment]
Screen Shot 2022-08-09 at 11 25 29 AM
Screen Shot 2022-08-09 at 11 25 38 AM

File: docs/typescript/introduction.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+id: introduction
+title: Introduction
+sidebar_label: Introduction
+---
+
+#### What is Typescript?
+
+Typescript is just a Superset of Javascript that brings additional features to javascript. It cannot be compiled by browser or any javascript compiler. It has its own compiler that converts typescript codes to acceptable javascript code.
```
Comment (samirsilwal) [inline docs/typescript/introduction.md:9]:
```suggestion
Typescript is just a superset of Javascript that brings additional features to javascript. It cannot be compiled by browser or any javascript compiler. It has its own compiler that converts typescript codes to acceptable javascript code.
```

**Comment by mesaugat** [comment]
Closing because we need more of a guideline than a tutorial format. Have to work on the guideline stuff.

**Comment by mesaugat** [comment]
Sad but thanks for the contribution @justsumant.

**Pull request closed without merging**